front/src/styles/scss/applications/stdcmV2/_home.scss

```diff
@@ -263,6 +266,44 @@
         }
       }
     }
+    .upgrade {
+      display: flex;
+      flex-direction: column;
+      margin: 0 2rem;
+      .text {
+       color: rgb(0, 0, 0);
+       margin-left: 0.5rem;
+       .title{
+       font-weight: 600;
+       font-size: 1.125rem;
+       padding-top: 1.563rem;
+       display: flex;
+       flex-direction: row;
+        .comment-icon {
```

Comment:
Add a display: flex, to get the same rendering as on sketch

Reply:
Done